dist/index.html

```diff
@@ -10,50 +10,52 @@
 <body>
   <section class="header">
     <img src="./images/around-the-world.png" class="globe" alt="spinning globe">
-    <div class="welcome">Globetrotter</div>
+    <div tabindex="0"class="welcome">Globetrotter</div>
     <img src="./images/around-the-world.png" class="globe" alt="spinning globe">
   </section> 
   <hr class="line">
-  <div class="welcome-user">Welcome</div>
+  <div tabindex="0" class="welcome-user">Welcome</div>
   <hr class="line">
    <section class="interface-wrapper">
       <section class="buttons">
         <div class="control-panel">Control Panel</div>
         <div class="button-side1">
-        <button class="control-button" id="past">Past</button>
-        <button class="control-button" id="present">Present</button>
+        <button class="control-button" tabindex="0" id="past">Past</button>
```

Comment:
Does your application (the buttons specifically) work without adding the tabindex? I think the button property should automatically tab through to buttons as it is a built in property. Maybe check before adding them in where you don't need to.